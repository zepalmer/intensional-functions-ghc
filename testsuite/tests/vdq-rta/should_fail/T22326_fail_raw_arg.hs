{-# LANGUAGE ExplicitNamespaces #-}
{-# LANGUAGE RequiredTypeArguments #-}

module T22326_fail_raw_arg where

f :: forall (a :: k) -> ()
f (type _) = ()

x = f (\_ -> _)