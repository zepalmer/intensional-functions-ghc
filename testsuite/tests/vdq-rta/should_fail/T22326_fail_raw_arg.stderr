
T22326_fail_raw_arg.hs:9:5: error: [GHC-29092]
    • Ill-formed type argument: \ _ -> _
      Expected a type expression introduced with the ‘type’ keyword.
    • In the expression: f (\ _ -> _)
      In an equation for ‘x’: x = f (\ _ -> _)
